Components/Table/Table.tsx

```diff
@@ -0,0 +1,50 @@
+import * as React from 'react';
+import {TableHeader} from './TableHeader';
+import {TableBody} from './TableBody';
+import {TableFooter} from './TableFooter';
+import styled from 'styled-components';
+
+export interface Column {
```

Comment:
@TravisSalad Just a suggestion, but since we're using `export *` in `index.tsx`, we may want to namespace interfaces or make them more obvious like `TableColumn` in order to ignore potential collisions.

Reply:
Looking through the code, it seems like most of the potential collisions have been mitigated already and I found the one case where it wasn't.  😆